website/content/api-docs/system/pprof.mdx

```diff
@@ -51,6 +89,12 @@ This endpoint returns stack traces of all current goroutines.
 | :----- | :--------------------- |
 | `GET`  | `/sys/pprof/goroutine` |
 
+### Parameters
+
+- `debug` `(int: 0)` - Specifies special arguments for the collection.
```

Comment:
Do we want to default to 2?

Reply:
I don't feel strongly one way or another.  But since this is a docs PR, if you think there's value in that, you should open a ticket.  Note however that most users aren't going to use this endpoint directly, they're going to use `vault debug`.  Also this would break backwards compat, though probably not a big deal here.

Reply:
Ah, I see. It would indeed break backwards compatibility. Leaning towards leaving it as-is.